awscli/paramfile.py

```diff
@@ -37,6 +37,10 @@
     'cloudformation.update-stack.stack-policy-url',
     'cloudformation.set-stack-policy.stack-policy-url',
 
+    'custom.cp.website-redirect',
```

Comment:
Cool. This change may be a little larger than expected (a change in the clidriver and custom commands). If you look further down in the list of black listed arguments, the ``s3api`` commands that use ``--website-redirect`` have the event listed as ``s3``...